src/main/java/uk/nhs/hee/tis/trainee/forms/service/LtftService.java

```diff
@@ -156,6 +156,7 @@ public Optional<LtftFormDto> updateLtftForm(UUID formId, LtftFormDto dto) {
           formId, traineeId);
       return Optional.empty();
     }
+    form.setCreated(existingForm.get().getCreated()); //explicitly set otherwise form saved as 'new'
```

Comment:
hackitty hackitty hack!

If the 'isNew()' logic uses a version field, then you don't need to set the created date, but (a) you do need to set the version from the saved form and (b) created is set to null.

Reply:
For want of a better approach, this looks okay. Googling around seems like there's no simple/OOTB solution for it, besides partial updates (which may be easier to implement with the new self-contained `content`, but something to pursue later).